src/utils/deep_copy.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright 2019-2021 ETH Zurich and the DaCe authors. All rights reserved.
```

Comment:
it's quite weird to put every method in its own .ts file. Could you colocate createelement/deepcopy/equals/lerp as one utils.ts file?